ooni/oonicli.py

```diff
@@ -1,10 +1,17 @@
 import sys
 import os
+import json
 import yaml
+import random
+import urlparse
 
 from twisted.python import usage
 from twisted.python.util import spewer
-from twisted.internet import defer
+from twisted.internet import defer, reactor, protocol, task
+
+import pika
+from pika import exceptions
+from pika.adapters import twisted_connection
```

Comment:
These imports should be moved down into the `runWithDaemonDirector` function since I think at this stage we don't want to make pika a hard dependency. The import of them should be wrapped in a `try: except ImportError` and print an error message saying that for this means of operation pika is required and should be installed with pip install pika.